shared-helpers/__tests__/Timeout.test.tsx

```diff
@@ -34,7 +40,7 @@ describe("<Timeout>", () => {
         <LoggedInUserIdleTimeout onTimeout={onTimeoutSpy} />
       </AuthContext.Provider>
     )
-    expect(createElementSpy).toHaveBeenCalledTimes(2)
+    expect(createElementSpy).toHaveBeenCalledTimes(1)
```

Comment:
Why did the number of calls change here?

Reply:
Oh, I wrote about it in the TODO comment above. Here:

> I don't think these tests mean anything now, as nothing is rendered in either case other than the wrapper div. We need a real way to test if the timeout dialog works but that would require a significant refactor beyond the scope of the current Seeds Dialog uptake task.

In other words, the test previously didn't really test the timeout behavior per se, but just tested if an additional element was created in the DOM because a user is logged in…not really what we actually want to test, and the switch to Seeds affected that brittle test.